File: exercicios/R/1145/1145.r
input = file('stdin','r')

count = as.integer(1)

line = strsplit(readLines(input,n=1), " ")

x = as.integer(line[[1]][1])
y = as.integer(line[[1]][2])

teste <-c()
for(i in 1:y){
    teste[i] = i
}

File: exercicios/JavaScript/1145/1145.js
var input = require('fs').readFileSync('stdin','utf8');
var lines = input.split('\n');

var line1 = lines.shift().split(' ');

var n = parseInt(line1[0]), qt = parseInt(line1[1]), count =1;
var teste = []

for(let i=1; i<=qt; i++){
    
    teste.push(i);
    if(count == n){
        console.log(teste.join(' '));
        count = 1;
        teste = []
    }else{
        count ++;
    }
}